Formula/libvterm.rb

```diff
@@ -0,0 +1,26 @@
+class Libvterm < Formula
+  desc "C99 library which implements a VT220 or xterm terminal emulator."
+  homepage "http://www.leonerd.org.uk/code/libvterm/"
+  url "http://www.leonerd.org.uk/code/libvterm/libvterm-0+bzr681.tar.gz"
+  sha256 "abea46d1b0b831dec2af5d582319635cece63d260f8298d9ccce7c1c2e62a6e8"
+
+  depends_on "libtool" => :build
+
+  def install
+    ENV.deparallelize
```

Comment:
This should be reported upstream with a link to that bug report included here. `make` not succeeding in parallel is a bug.